lib/linguist/languages.yml

```diff
@@ -6946,7 +6946,6 @@ VBA:
   - ".bas"
   - ".cls"
   - ".frm"
-  - ".frx"
```

Comment:
Whoops. You don't want to remove this extension. You don't want to remove the `.frm` extension either.

Reply:
`.frx` should not have been added in the first place since it is a binary data file and not VB6 code. And the format is proprietary.

> An .FRX file is a binary data file used to store binary information for the form (.FRM) file with the same file name.

> 

> MORE INFORMATION
> Some controls, such as the PictureBox and Image controls, have properties that use binary data as their values. Visual Basic stores the binary data in an .FRX file with the same file name as the Form file. The .FRX file is a proprietary format.
> 
> If your project contains .FRX files, you should include these files with the corresponding .FRM files. A form containing binary information but lacking the associated .FRX files will not function correctly.

[source](https://www.vbforums.com/showthread.php?115463-What-is-an-frx-file&p=617730&viewfull=1#post617730)